### ISSUE TITLE
Modal doesn't close after adding a new encounter + encounter form spacing

### ISSUE THREAD

**Issue by Trodrige** [LibreHealthIO/LibreEHR#837]
Go to Popups->Issues. Click on Add Encounter. The form that shows, the input fields in the middle should have some spacing. Like in the "Add Issue" form. The "Add" button should have some space form the textarea below it.
Try adding a new issue by clicking "Add" button on the "New encounter form". Save the issue. Notice that modal doesn't disappear after saving the issue.

**Comment by Trodrige** [comment]
@aethelwulffe what do you think about the spacing here? Should the form be made compact or make use of the available space the modal has.

**Comment by Jdew192837** [comment]
Also, we should have the option to delete an issue right? @Trodrige and @aethelwulffe

**Comment by Trodrige** [comment]
@Jdew192837 yes. That is if you mean deleting the issue when the user realises they added the wrong one. You're smart, you noticed this. Great, delete issue? I say yes. Go for it!!!

**Comment by teryhill** [comment]
If the issue has any information or is not an instant oh boy I made a mistake. The it will be an issue deleting it. Say some one just wants to mess up the clinic and the patient has a fatal allergy to aspirin, and the doctor give the patient 2 in the office for their headache.

**Comment by Trodrige** [comment]
@teryhill I don't get what you mean.

**Comment by Trodrige** [comment]
@Jdew192837 a plus here. "Begin date" and "End date" should be on same line. That is on both "Add issue" forms.

**Comment by teryhill** [comment]
The physicians rely on the information there for a history, If you give the ability to delete it cannot be an "Old" issue. @Trodrige

**Comment by Trodrige** [comment]
What if a wrong title or begin date is entered while adding an issue? I don't know if I'm missing something, but the user entering the information should be able to correct  it.

**Comment by teryhill** [comment]
To me if you could Inactivate then it would be better than deleting them. That way if some one was on the wrong patient and inactivated the issues they could be gotten back. I am not fond of hard deletes.

**Comment by aethelwulffe** [comment]
A "Mark as deleted" or a "Move to patient X" would be better.  Errors DO happen, and they can be just as bad as having something deleted...or worse.
@Jdew192837 I have no idea how to make the modal window go away.  Sorry.  I would be on Stack Exchange trying to find the answer myself.

**Comment by muarachmann** [comment]
@Trodrige  , @aethelwulffe , @teryhill  suceeded in solving this problem. I could save an issue and notify that it was successfully saved and make the pop up disappear

**Comment by Trodrige** [comment]
@muarachmann do a PR so I can test.

**Comment by muarachmann** [comment]
ok doing that

**Comment by muarachmann** [comment]
@teryhill  here is the issue am talking about

> Go to Popups->Issues. Click on Add Encounter. The form that shows, the input fields in the middle should have some spacing. Like in the "Add Issue" form. The "Add" button should have some space form the textarea below it.

**Comment by aethelwulffe** [comment]
@muarachmann I don't feel that it needs space myself.  As a matter of fact, a great many things are wasting viewable area IMO.  The white-space in criteria selection and controls heavily impacts the viewability of the data you are trying to get to; especially on smaller screens.  I get that the modal window is of a certain size and all, but even so, we should keep things tight (in my humble opinion).  Better would be a control padding configuration setting in the theme color-picker, and have all controls look to that instead of a hard setting or bootstrap or the like.
